firerunner/src/main.rs

```diff
@@ -39,12 +56,12 @@ fn main() {
                 .value_name("CMD_LINE")
                 .takes_value(true)
                 .required(false)
-                .default_value("console=ttyS0 reboot=k panic=1 pci=off")
+                .default_value("console=ttyS0 reboot=k panic=1 pci=off nokaslr")
```

Comment:
Is `nokaslr` necessary? Is it relevant to creating snapshots?

Reply:
oops, no. I forgot to remove it.